



3
1_R.rb:3:in `suma': wrong number of arguments (given 1, expected 2) (ArgumentError)